gator/invoke.py

```diff
@@ -11,6 +11,9 @@
 from gator import run
 from gator import util
 
+# import snoop
+# snoop.install(color="rrt")
```

Comment:
Thanks, this is a good point with which I agree! I fixed this in the file that you mentioned and in all of the other files that had this commented-out code. I also added instructions to the `CONTRIBUTING.md` file so that there is documentation about how to perform Snoop-based debugging, which I think could help many of our contributors.

I'm marking this issue as resolved! Let me know if you have other concerns about it, okay?